homeassistant/components/avocent_dpdu/entity.py

```diff
@@ -0,0 +1,11 @@
+"""Entity classes for the Avocent Direct PDU integration."""
+
+from __future__ import annotations
+
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .coordinator import AvocentDpduDataUpdateCoordinator
+
+
+class OutletEntity(CoordinatorEntity[AvocentDpduDataUpdateCoordinator]):
+    """Define an Outlet entity."""
```

Comment:
You can add `_attr_has_entity_name = True` here and the device info would be nice as well